docs/CMakeLists.txt

```diff
@@ -73,14 +73,13 @@ set(CPACK_COMPONENTS_ALL ${CPACK_COMPONENTS_ALL} doxygen)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/modules" ${CMAKE_MODULE_PATH})
 find_package(Sphinx REQUIRED)
 
-set(DOCS_BUILDER html)
```

Comment:
I think the error comes from here. Do not remove the `DOCS_BUILDER` variable.

Reply:
One to many lines 🤦🏼